tests/test_write_anything.py

```diff
@@ -40,30 +43,37 @@
     "SPDXJsonExample2.2.json-tag",
 }
 
+
 @pytest.mark.parametrize("out_format", ['rdf', 'yaml', 'xml', 'json', 'tag'])
-@pytest.mark.parametrize("in_file", test_files, ids=lambda x: os.path.basename(x))
-def test_write_anything(in_file, out_format, tmpdir):
+@pytest.mark.parametrize("in_file", test_files_json_yaml_xml_tag, ids=lambda x: os.path.basename(x))
+def test_write_anything_json_yaml_xml_tv(in_file, out_format, tmpdir):
     in_basename = os.path.basename(in_file)
-    if in_basename == "SPDXSBOMExample.spdx.yml" or in_basename == "SPDXSBOMExample.tag" or in_basename == "SPDXJsonExample2.2.json":
+    if in_basename == "SPDXSBOMExample.spdx.yml" or in_basename == "SPDXJsonExample2.2.json" or in_basename == "SPDXSBOMExample.tag":
         # conversion of spdx2.2 is not yet done
         return
-    doc, error = parse_anything.parse_file(in_file)
+    write_anything_test(in_basename, in_file, out_format, tmpdir)
+
+
+@pytest.mark.parametrize("out_format", ['rdf'])
+@pytest.mark.parametrize("in_file", test_files_rdf, ids=lambda x: os.path.basename(x))
+def test_write_anything_rdf_tag(in_file, out_format, tmpdir):
```

Comment:
`test_write_anything_rdf`

Reply:
Also, maybe we should add a comment here explaining why the in/out formats are split like this

Reply:
done